docker-compose.yaml

```diff
@@ -0,0 +1,69 @@
+version: '3.8'
+
+services:
+  react-app:
+    build:
+      context: ./
+      dockerfile: Dockerfile.react # Creates new image for react app
+    expose:
+      - "8080" # Expose port for other services
+    ports:
+      - "8080:80"
+    networks:
+      - nginx_default
+
+  solid-server:
+    image: solidproject/community-server:latest # Use updated image or create using Dockerfile
+    environment:
+      - BASE_URL=https://css.opencommons.net
```

Comment:
The [environment variables section of the CSS readme](https://github.com/CommunitySolidServer/CommunitySolidServer#%EF%B8%8F-environment-variables) says that the environment variables used by Community solid server need to be prefixed with `CSS_`, so this should be `CSS_BASE_URL`.